src/main/java/org/opensearch/knn/index/codec/KNN990Codec/NativeEngines990KnnVectorsReader.java

```diff
@@ -159,4 +194,31 @@ public void close() throws IOException {
     public long ramBytesUsed() {
         return flatVectorsReader.ramBytesUsed();
     }
+
+    private void primeQuantizationStateCache() throws IOException {
```

Comment:
Can you add a comment explaining what we are doing in this method

Reply:
So the method is priming the quantization state cache and that was the intent of the name change